### PR TITLE
Small improvements to chemdispenser UI

### LINKED ISSUE #3961: [Feature Request] Have chemdispensers remember custom amounts
It's useful to be able to dispense 15u at a time, but since the button clears when you select another option, you have to click Custom and type 15 every time. It'd be more useful if the "Custom" button remembered whatever amount you gave it, and clicking it twice or having a small "edit" button next to it or whatever to modify the custom amount.

### PR DESCRIPTION
**Pull request by 9600bauds**
Fixes #3961, the "custom" field now remembers the last value entered.
Also adds 15 and 25 to the possible amounts to choose from and fixes the 50 button being wonky.
Rogue clamp

### CODE REVIEW

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 1]
```diff
@@ -183,17 +184,15 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
...
+				useramount = amount
 				custom = 1
 		else
 			custom = 0
 			amount = round(text2num(href_list["amount"]), 5) // round to nearest 5
-		if (amount < 5) // Since the user can actually type the commands himself, some sanity checking
-			amount = 5
-		if (amount > 100)
-			amount = 100
+		amount = Clamp(amount, 5, 100) // Since the user can actually type the commands himself, some sanity checking
```
Comment (ComicIronic) [inline code/modules/reagents/Chemistry-Machinery.dm]: You should add the useramount setting after this, for sanity.
Try

    amount = Clamp(amount, 5, 100)
    if(custom)
        useramount = amount

**Comment by clusterfack** [comment]
You made sure it still maintains the 3 rows and all that, right?

**Comment by 9600bauds** [comment]
Yes, I think the issue with one of the buttons "running off" was that the window wasn't made larger when Tungsten was added to the dispenser.
ss 2015-04-18 at 03 16 11

**Comment by clusterfack** [comment]
Alright that's all I cared about particularly, nice work

**Pull request merged**